docs/server-instructions.md

```diff
@@ -111,3 +112,16 @@ Add the path to the JVM agent to the `jvm.options` file:
    -javaagent:/path/to/splunk-otel-javaagent.jar
    ```
 - Save the file and restart the server.
+
+## WebSphere Traditional
+
+1. Open the WebSphere Admin Console
+2. Select **Servers > Server type > WebSphere application servers**.
+3. Click on the desired server.
+4. Select **Java and Process Management > Process Definition**.
+5. Click on **Java Virtual Machine**.
+6. In the **Generic JVM arguments**, enter the path to `splunk-otel-javaagent.jar`:
+   ```
+   -javaagent:/path/to/splunk-otel-javaagent.jar
+   ```
+7. Press **OK**. When asked, save the master configuration and restart the server.
```

Comment:
```suggestion
7. Click **OK**. When asked, save the master configuration and restart the server.
```

Reply:
This is the preferred path? Any programmatic equivalent users might prefer?

Reply:
Websphere can be programmatically configured using the `wsadmin` tool that accepts jython and jacl scripts. I would prefer not to describe how exactly this is done because it isn't that simple. Anyway websphere users should be accustomed with hardship. Those that already know about `wsadmin` are able to figure this out themselves.